docs/design_guide.md

```diff
@@ -42,63 +35,24 @@ election.
 ### Audit
 
 An abstract class to define a *general* audit. It will define attributes and
-parameters common to all audits:
-
-- risk limit (alpha)
-- beta
-- maximum number of ballots to draw during audit
-- object containing information about election/contest to be audited
-  - could be `Election` with set of `Contest`s
-  - could be a single `Contest`
-- replacement
-
-*Note:* Some of the above will be provided by the users for each unique audit,
-but others might be defined by developers in the audit sub-classes and be
-constant values for each unique audit of that type.
-
-Example pseudocode:
-
-```python
-class audit:
-  alpha: float
-  beta: float
-  max_num_ballots: int
-  replacement: bool
-
-  def __init__(self, alpha, election, ...):
-    self.alpha = alpha
-    self.election = election
-    ...
-```
-
+methods common to all audits. In the future, it will provide frameworks for an
+interactive audit execution and a 'bulk' execution given a data set.
 
-**Methods** \**incomplete*
+For the most up-to-date version of the `Audit` class, look
+[here](https://r2b2.readthedocs.io/en/latest/autoapi/r2b2/audit/index.html)
```

Comment:
I try to avoid any use of link text like "here" since it says so little.
Link text like `r2b2/audit/index.html` (or without the index.html) would seem better.